docs/_src/language/basic.md

```diff
@@ -60,48 +51,65 @@ explore 'malloy-data.faa.airports'
 ```
 
 ## Everything has a Name
-In Malloy, all output fields are named.  Malloy uses `<fieldname> is <expression>` instead of SQL's `<expression> as <fieldname>` so that the object being named comes first.  Having the output column name written first makes imagining the structure the output table easier.
 
-## Expressions are similar to SQL
-Mostly, your SQL expressions just work.  In Malloy, you can use almost all the same functions you can use in SQL (there are some new ones for convienece).
+In Malloy, all output fields are named. This means that any time a query includes a
+calculation or agregation, it must be aliased.
+
+```malloy
+--! {"isRunnable": true, "runMode": "auto", "isPaginationEnabled": true}
+explore 'malloy-data.faa.airports' | reduce
+  max_elevation is max(elevation)
+```
+
+Malloy uses `name is value` instead of SQL's `value as name` so the object being named comes first. Having the output column name written first makes imagining the structure the output table easier.
 
-The result of this is fairly straightforward based on a knowledge of SQL.
+Columns from a table and fields defined in an explore already have names, and can be referenced directly.
 
 ```malloy
 --! {"isRunnable": true, "runMode": "auto", "isPaginationEnabled": true}
-explore 'malloy-data.faa.airports'
-| reduce
-  state_and_county is concat(state,', ', county )
+explore 'malloy-data.faa.airports' | project
+  full_name,
+  elevation
+```
+
+## Expressions
+
+Many SQL expressions will work unchanged in Malloy, and almost all functions available in Standard SQL are usable in Malloy as well. This makes expressions fairly straightforward to understand given a knowledge of SQL.
+
+```malloy
+--! {"isRunnable": true, "runMode": "auto", "isPaginationEnabled": true, "size": "large"}
+explore 'malloy-data.faa.airports' | reduce
+  county_and_state is concat(county, ', ', state)
   airport_count is count()
   max_elevation is max(elevation)
   min_elevation is min(elevation)
   avg_elevation is avg(elevation)
 ```
-## Malloy data types are more simple than SQL
-String, number, boolean are the basic types.
 
-## Expressions can be modeled and re-used
-One of the main benefits of Malloy is to be able to build calculations you might want to use
-again into a data model.  In the example below, we create an *explore* object named `airports` and
-add calculations for `state_and_county` and `airport_count`.
+Malloy has fewer variations of common data types than Standard SQL: `string`, `number`, `boolean`, `date`, and `timestamp` are the most common.
```

Comment:
I was also trying to get across the point that the types are simpler: rather than `INTEGER`, `INT64`, `FLOAT`, and `FLOAT64`, there's just `number`. Not really sure how to get that across nicely though. LMK if you have thoughts.